types/google-spreadsheet/index.d.ts

```diff
@@ -1168,8 +1168,9 @@ export class GoogleSpreadsheet implements SpreadsheetBasicProperties {
     /**
      * @description
      * load basic Spreadsheet document properties and child worksheets
+     * Pass in (optional) truthy parameter to also load grid data.
```

Comment:
I'd use JSDoc to document the parameter

```suggestion
     * 
     * @param includeCells whether to also load grid data
```